### ISSUE TITLE
Normalize reducer RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by xiaobuu** [erikras/redux-form#363]
It worked before, but was broken after I update the package.
my form is like 
```
{
      startTime: now.format('HH:mm'),
      dueTime: then.format('HH:mm'),
      reward: '1',
      description: '',
      secret: '',
      tags:[],
}
```
my from reducer is decorated by both normalize reducer and plugin reducer.
I have a normalize function for the startTime and dueTime fields.

the new error is happening in getValuesFromState.js
```
else {
        accumulator[key] = getValuesFromState(field);
      }
``` 

after adding a condition manually it works again
```
else  if (typeof field !== 'string')  {
         accumulator[key] = getValuesFromState(field);
}
```

Am I using the reducer wrong, or the new getValuesFromState is buggy?

**Comment by erikras** [comment]
Looks like a bug. Will investigate. :mag:

**Comment by erikras** [comment]
[Published fix in `v3.1.7`](https://github.com/erikras/redux-form/releases/tag/v3.1.7).
